Formula/qt.rb

```diff
@@ -24,39 +24,35 @@ class Qt < Formula
     sha256 "25c4a693c787860b090685ac5cbeea18128d4d6361eed5b1bfed1b16ff6e4494" => :mojave
   end
 
-  keg_only "Qt 5 has CMake issues when linked"
+  keg_only "Qt has CMake issues when linked"
 
+  depends_on "cmake" => [:build, :test]
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+
   depends_on xcode: :build
-  depends_on macos: :sierra
+  depends_on macos: :mojave
 
   uses_from_macos "bison"
   uses_from_macos "flex"
+  uses_from_macos "icu4c"
   uses_from_macos "sqlite"
+  uses_from_macos "zlib"
```

Comment:
Add these macos dependencies
  uses_from_macos "cups"
  uses_from_macos "krb5"
  uses_from_macos "perl"

Reply:
Why?

Reply:
During configuring the qt, qt need run a perl script, and use CMake detect dependencies, Qt Network module use GSS feature (in fact... krb5 is not enough, it need entire GSS framework)